.pull-review

```diff
@@ -0,0 +1,63 @@
+---
+# pull-review config version (required)
+version: 1
```

Comment:
Does this version number have to be incremented each time the configuration is changed or does it refer to the version of the software?

Reply:
It refers to the version of pull-review. So we don't have to change it when modifying the configuration file.